### DISCUSSION TITLE
连不上trojan

### DISCUSSION THREAD

**Discussion by gallexy** [yanue/V2rayU#1042] [Q&A]
自己搭了trojan, clashx , v2rayNG都能连, 但v2rayU连不上, 是不是不支持?

**Comment by terryleeorz** [comment]
我也有类似问题，自己搭了v2ray server， 安卓v2rayNG可以连，mac侧的v2rayU连不上

**Comment by ctra-wang** [comment]
https://github.com/yanue/V2rayU/discussions/1152
